orchest-sdk/python/orchest/errors.py

```diff
@@ -57,3 +93,13 @@ class DataInvalidNameError(Error):
     """Data has an invalid name"""
 
     pass
+
+
+class InvalidMetaDataError(Error):
+    """Metadata is invalid
+
+    For metadata that is missing the separator, that does not contain
+    the right amount of elements, or that has invalid content.
+    """
```

Comment:
```suggestion
    """Metadata is invalid."""
```
Not sure the second paragraph of the error has an added benefit since when using the `InvalidMetaDataError` the message already explain why the metadata is valid, e.g. it is missing the separator.

Reply:
Agree